<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GrayHistogramDialog</class>
 <widget class="QDialog" name="GrayHistogramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBoxInfo">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>40</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>统计方式</string>
   </property>
   <widget class="QRadioButton" name="radioButtonPic">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>全图统计</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonBin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>二值化数据统计</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxHW">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>190</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>行/高统计</string>
   </property>
   <widget class="QRadioButton" name="radioButtonColumn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>按列统计</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonRow">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>按行统计</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxPix">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>340</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>像素统计方式</string>
   </property>
   <widget class="QRadioButton" name="radioButtonWhhiltePix">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>统计白像素</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonBlackPix">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>统计黑像素</string>
    </property>
   </widget>
  </widget>
  <widget class="LabelGrayHistogram" name="labelPic">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>680</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>680</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelGrayHistogram</class>
   <extends>QLabel</extends>
   <header>GrayHistogram/Roi/LabelGrayHistogram.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
